src/services/utility.service.ts

```diff
@@ -57,24 +86,33 @@ class UtilityService {
    * @param inputText Input text for search
    * @param setSearchResults Function for setting state of search results
    * @param setDisplayValue Function for setting display text of search
+   * @param currentLocation Current location of User
    *
    * TODO: Add current location as a searchable element by passing it as
    * a parameter and including it in the search.
    */
   public updateSearchResults = (
     inputText: string,
     setSearchResults: (locations: SearchResult[]) => void,
-    setLocationDisplay: (display: string) => void
+    setLocationDisplay: (display: string) => void,
+    currentLocation: Location
   ) => {
-    const locations: SearchResult[] = this.locationsToSearch.filter(
-      (location) => {
-        return (
-          location.searchName.toLowerCase().search(inputText.toLowerCase()) !==
-          -1
-        );
-      }
-    );
+    let locations: SearchResult[] = [];
+    locations = this.locationsToSearch.filter((location) => {
```

Comment:
This was at my request. It removes the need for the ternary operator below which should have been removed in the PR but I forgot to mention it. This fixes a bug where the location array wasn't being reset to empty every time the function was called so it would always have 5 values in it even if the text only matched 3.

Reply:
good catch, fixed